docs/dev-concepts/faq.md

```diff
@@ -179,6 +187,10 @@ The XMTP SDK currently requires you to use [ethers](https://ethers.org/) or anot
 
 ### Why does my app request a new signature for XMTP when I refresh my browser?
 
-The XMTP client provided by the SDK requires a user's signature in order to decrypt their XMTP message encryption keys. This process must be repeated when starting a new session since there is no secure place in the browser to persist decrypted keys. Based on developer and community feedback, we are researching more robust approaches to secure key management.
+The XMTP client provided by the SDK requires a user's signature to decrypt their keys needed for invitation and message encryption. This process must be repeated when starting a new session since there is no secure place in the browser to persist decrypted keys. 
+
+If you are using the JavaScript client SDK, you might consider [manually handling private key storage](/docs/client-sdk/javascript/tutorials/quickstart#manually-handling-private-key-storage), but only with the understanding that this approach **requires the utmost care**.
+
```

Comment:
@neekolas - what do you think of this update to the FAQ? Is this something we might want to surface here - or is it better that it remain in the README context only?